listeners.go

```diff
@@ -262,43 +262,73 @@ var (
 	listenersMu sync.Mutex
 )
 
+const maxPortSpan = 65535
+
+type Listener struct {
+	Network  string
+	Host     string
+	FromPort uint
+	ToPort   uint
```

Comment:
I might be a bit pendantic, but what if we called these `StartPort` and `EndPort` instead? (it's not really "going" from one port to another, even though yes in some sense it's a range or span, but I tend to think of From -> To when there is interaction between those things; these ports don't interact)

(Most editors should have a nice Rename Symbol option so that you don't have to replace all occurrences manually)